index.html

```diff
@@ -14,6 +16,96 @@
+  <div class="container">
+    <header>
+      <nav>
+        <a href="#">Résumé</a>
+        <a href="#">About</a>
+        <a href="#">Portfolio</a>
+        <a href="#">Contact</a>
+      </nav>
+    </header>
+
+    <div class="main-content">
+      <div class="treepic">
+        <img src="https://dl.kraken.io/web/d8cc605bf9d78756364dc55b64f66aa0/treeshadow.jpg" alt="treeimage">
+      </div>
+      <div class="nameBox">
+        <h1>ALISON LAW - Web Dev</h1>
+        <h3>Technology is ...</h3>
+        <br>
+        <hr>
+        <br>
+        <p>Occasionally I have time to work on my side projets. If you're interested, let me know by signing up on this
+          website.</p>
+        <button>Let's Team Up</button>
+      </div>
+    </div>
+
+    <div class="middle">
+      <div class="statement">
+        <H2>Listen Up...</H2>
+        <br>
+        <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ullam dolorum illum vitae explicabo ea
+          consequuntur, minima fuga nostrum ipsa facere similique sint id quaerat a impedit! Qui accusantium dolores
+          cum? Lorem ipsum dolor sit amet consectetur adipisicing elit. Eos reprehenderit vitae illum, cum repellendus odio necessitatibus nisi neque ut ipsum dicta placeat suscipit numquam iste voluptatum odit quis? Fuga, officia!
+          Lorem ipsum, dolor sit amet consectetur adipisicing elit. Excepturi, iure non velit assumenda ipsam ab numquam voluptatibus nam, quos voluptatum quisquam rerum possimus tempore distinctio error ducimus sint molestiae ullam?
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Laboriosam numquam optio, a debitis doloribus porro velit. Ipsam cumque, tempore provident quod vitae nulla harum, dolor quam a voluptatibus ipsa voluptate.
+        </p>
+      </div>
+
+      <div class="stuff">
+        <h2>Stuff I'm good at:</h2>
+        <br>
+        <ul>
+          <li>Interior design</li>
+          <li>Rowing</li>
+          <li>Tango</li>
+          <li>Chilling</li>
+          <li>Something super impressive</li>
+         
+        </ul>
+      </div>
+    </div>
+
+    <div class="bottomSection">
+      <h2>Work History</h2>
+      <br>
+      <table>
+          <tr>
+            <th class="odd">Yrs</th>
+            <th class="odd">Company</th>
+            <th class="odd">Role</th>
+          </tr>
+          <tr>
+            <td class="even">1</td> </td>
+            <td class="even">Lambda School </td>
+            <td class="even">Student</td>
+            <tr>
+            <td class="odd">7</td>
+            <td class="odd">Cambridge University Hospital NHS Trust </td>
+            <td class="odd">Epic Application analyst</td>
+          </tr>
+          <tr>
+              <td class="even">14</td>
+              <td class="even">Cambridge University Hospital NHS Trust </td>
+              <td class="even">Physiotherapist (Musculoskeletal)</td>
+            </tr>
+          </tr>
+        </table>
+    </div>
```

Comment:
Great job on using the table tags! I would have liked to see them styled similar to the design image. Try giving even rows an offsetting color.

Reply:
Added grey and whith stries using nth-child pseudo property